ui/ui.go

```diff
@@ -0,0 +1,93 @@
+package ui
+
+import (
+	"fmt"
+	"html/template"
+	"log"
+	"net/http"
+
+	"github.com/ImageWare/TLSential/certificate"
+	"github.com/ImageWare/TLSential/model"
+	"github.com/gorilla/mux"
+)
+
+const localStaticDir = "./static"
```

Comment:
Should probably add a TODO or Issue to make this agnostic to Current Working Directory so that it doesn't matter what directory the server is started from.

Reply:
Whoops. One is on line 35